typings/vscode-mssql.d.ts

```diff
@@ -449,7 +449,7 @@ declare module 'vscode-mssql' {
 		generateScript(operationId: string, targetServerName: string, targetDatabaseName: string, taskExecutionMode: TaskExecutionMode): Thenable<ResultStatus>;
 		publishDatabaseChanges(operationId: string, targetServerName: string, targetDatabaseName: string, taskExecutionMode: TaskExecutionMode): Thenable<ResultStatus>;
 		publishProjectChanges(operationId: string, targetProjectPath: string, targetFolderStructure: ExtractTarget, taskExecutionMode: TaskExecutionMode): Thenable<SchemaComparePublishProjectResult>;
-		getDefaultOptions(): Thenable<SchemaCompareOptionsResult>;
+		schemaCompareGetDefaultOptions(): Thenable<SchemaCompareOptionsResult>;
```

Comment:
Maybe document the methods used by projects extension so we are more careful about changing them in the future.

Reply:
I added this comment block above the declared interface: 
![image](https://github.com/user-attachments/assets/e20ad68a-0ddf-43d3-8c47-0c76af94abf3)